doc/docs.md

```diff
@@ -1586,10 +1630,10 @@ Let's create a simple program `test.cpp` first:
 #include <iostream>
 
 int main() {
-        std::vector<std::string> s;
```

Comment:
what is this? this should not be changed.

Reply:
my fault with replacing things, i will change it back to table and this error too